pyhap/accessory_driver.py

```diff
@@ -510,6 +536,7 @@ def load(self):
         with open(self.persist_file, "r") as file_handle:
             self.encoder.load_into(file_handle, self.state)
 
+    @callback
     def pair(self, client_uuid, client_public):
```

Comment:
I didn't rename this to `async_pair` since it may be overridden in a child class